src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -1750,7 +1750,11 @@ private Symbol GetDeclaredMember(NamespaceOrTypeSymbol container, TextSpan decla
         /// <summary>
         /// Finds the member in the containing symbol which is inside the given declaration span.
         /// </summary>
-        private Symbol GetDeclaredMember(NamespaceOrTypeSymbol container, TextSpan declarationSpan, string name = null)
+        /// <param name="isKnownNamespace"><see langword="true"/> if the result is known to be a
+        /// <see cref="NamespaceSymbol"/> (e.g. when the caller is <see cref="GetDeclaredNamespace(BaseNamespaceDeclarationSyntax)"/>;
+        /// otherwise, <see langword="false"/> if the symbol kind is either unknown or known to not be a
+        /// <see cref="NamespaceSymbol"/>.</param>
+        private Symbol GetDeclaredMember(NamespaceOrTypeSymbol container, TextSpan declarationSpan, bool isKnownNamespace, string name = null)
```

Comment:
>isKnownNamespace [](http://example.com/codeflow?start=105&length=16)

I think the name is somewhat confusing. Consider using "isKnownToBeANamespace" instead.

Reply:
⏱️ Renaming